02-spatial-data.Rmd

```diff
@@ -774,16 +769,15 @@ r_stack <- stack(raster_in_memory, raster_on_disk)
 r_stack
 ```
 
-Due to its properties, `RasterBrick` objects should be processed in a shorter time than `RasterStack`.
-Additionally, operations on `RasterBrick` and `RasterStack` objects will typically return `RasterBrick`.
-On the other hand, `RasterStack` give more flexibility, as a single object could be related to data stored in a memory and on disk in the same time.
-`RasterBrick` objects could be stored only in a memory or on disk.
+Another difference is that the processing time for `RasterBrick` objects should be usually shorter than for `RasterStack` objects.
+Note that operations on `RasterBrick` and `RasterStack` objects will typically return a `RasterBrick`.
 
 ## Coordinate Reference Systems
```

Comment:
Great to have this new raster content - thanks for that @Nowosad and for the comments @jannes-m. My vision is that we make a few basic comments in the next section before a longer section in Chapter 6. Sound good?